code/workspaces/infrastructure-api/src/models/centralAssetRepo.model.js

```diff
@@ -0,0 +1,28 @@
+import mongoose from 'mongoose';
+import { v4 as uuid } from 'uuid';
+import { assetTypes } from 'common/src/config/catalogue';
+
+const modelName = 'CentralAssetMetadata';
+
+const possibleTypeValues = () => assetTypes();
+const possibleVisibleValues = () => ['public', 'byProject'];
```

Comment:
I'd capitalise these too, like assetTypes

Reply:
Good shout, I'll make the change